benchmarks/serialization_libraries/compare.sh

```diff
@@ -0,0 +1,222 @@
+#   },                                                              },
+#   "type": "comments"                                              "type": "comments"
+# }                                                               }
+# {                                                               {
+#   "attributes": {                                                 "attributes": {
+#     "author": "me",                                                 "author": "me",
+#     "comment": "nice blog"                                          "comment": "nice blog"
+#   },                                                              },
+#   "id": "6",                                                      "id": "6",
+#   "relationships": {                                              "relationships": {
+#     "post": {                                                       "post": {
+#       "meta": {                                               |       "data": {
+#         "included": false                                     |         "id": "5",
+#                                                               >         "type": "posts"
+#       }                                                               }
+#     }                                                               }
+#   },                                                              },
+#   "type": "comments"                                              "type": "comments"
+# }                                                               }
+# {                                                               {
+#   "attributes": {                                                 "attributes": {
+#     "author": "me",                                                 "author": "me",
+#     "comment": "nice blog"                                          "comment": "nice blog"
+#   },                                                              },
+#   "id": "7",                                                      "id": "7",
+#   "relationships": {                                              "relationships": {
+#     "post": {                                                       "post": {
+#       "meta": {                                               |       "data": {
+#         "included": false                                     |         "id": "5",
+#                                                               >         "type": "posts"
+#       }                                                               }
+#     }                                                               }
+#   },                                                              },
+#   "type": "comments"                                              "type": "comments"
+# }                                                               }
+# -------------------------------------included types: ams
+# "comments"
+# "posts"
+# -------------------------------------included types: jsonapi_rb
+# "comments"
+# "posts"
```

Comment:
It was much simpler to compare comments vs comments and posts vs posts than included vs included.  Since I'm whitelisting the included types for comparison, this validates that they both include the same types of only comments and posts